packages/fluentui/react-northstar/src/components/Tree/Tree.tsx

```diff
@@ -398,7 +398,7 @@ class Tree extends AutoControlledComponent<WithAsProp<TreeProps>, TreeState> {
 
     const renderItems = (items: TreeItemProps[], level = 1, parent?: string): React.ReactElement[] => {
       return items.reduce((renderedItems: React.ReactElement[], item: TreeItemProps, index: number) => {
-        const { id } = item;
+        const { id = _.uniqueId() } = item;
```

Comment:
@silviuaavram is it a correct fix? As I remember `id`s should be stable as `key`s in React.

Reply:
In this case id is just internal in the component, the alternative here would be to use `item` as `id` when `typeof item === 'string'`. But I am assuming will make no difference since this id is not being used like key to keep track.

Reply:
```suggestion
        const id = typeof item === 'string' ? item : item.id;
```

This would be the alternative

Reply:
`activeItemIds` rely on `id`s. If they will change on next render some items will accidentally close